package.json

```diff
@@ -61,15 +61,17 @@
     "prettier:check": "prettier --check ./src",
     "prettier:fix": "prettier --write ./src",
     "server": "node server.js",
-    "start": "cross-env REACT_APP_USE_DB_EMULATORS=false react-scripts start",
+    "start": "run-p --race dev:firebase dev:react",
```

Comment:
Still does the same thing. Only now it uses firebase to serve the react app. this is really cool cause now if we switch between production and staging on our local it will automatically switch to that DB. gonna make solving prod issues so much easier